gapic-generator-cloud/templates/cloud/client/_class.erb

```diff
@@ -11,9 +11,17 @@ require "google/gax"
 require "<%= service.gem.version_require %>"
 require "<%= service.service_proto_require %>"
 require "<%= service.credentials_class_require %>"
+<%- if service.paths? -%>
+require "<%= service.paths_require %>"
+<%- end -%>
 <% end %>
 # Service that implements <%= service.name %> API.
 class <%= service.client_class_name %>
+<%- if service.paths? -%>
+  include <%= service.paths_name %>
+  extend <%= service.paths_name %>
```

Comment:
I think we should only include the module. Since we are changing the Client to be `Service::Client` rather than `ServiceClient` these methods won't be backwards compatible if we make them class methods on the client. Furthermore I don't think theres much benefit to exposing the methods on `Service::Client` when we are already exposing them on `Service::Paths` since the amount of modules the user has to go through is the same for both uses. I'd prefer to remove extend to try and reduce the size of the interface.

Reply:
I can support removing the `extend`. I added it because it is now things are done now, but I agree it is easier to reuse as a module than it ever was as class methods on the client.

![break-all-the=things](https://user-images.githubusercontent.com/730/55254165-c89ff980-521c-11e9-9e58-3fa989625b95.jpg)